requirements.txt

```diff
@@ -2,3 +2,4 @@ boto3>=1.4.0
 cached_property
 python-keystoneclient>=1.8.1
 python-swiftclient
+futures>=3.0.5; python_version < '3.0'
```

Comment:
agreed, I think it's possible to do this without futures and rely on test suite to catch it.